platform/branding-api/pom.xml

```diff
@@ -47,6 +46,10 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.name}</Bundle-Name>
+                        <Import-Package>
+                            !org.springframework.core.io.support,
```

Comment:
❓ Do we know why this had to be added? It seems like nothing changed for this module.

Reply:
I don't know why. I encountered missing requirement issues, and this was just one solution that I found.